packages/ui-alerts/src/Alert/index.tsx

```diff
@@ -23,7 +23,7 @@
  */
 /** @jsx jsx */
 import React, { Component } from 'react'
-import ReactDOM from 'react-dom'
+import { createRoot, Root } from 'react-dom/client'
```

Comment:
I ended up refactoring the `Alert` component to not to use `ReactDOM.render`/`React.createRoot.render` API, instead it will render the screen reader content using `React.Portal`. Doing this made the component much easier to understand without any breaking changes.